docker-compose.yml

```diff
@@ -46,7 +55,7 @@ services:
     image: tootsuite/mastodon
     restart: always
     env_file: .env.production
-    command: bash -c "rm -f /mastodon/tmp/pids/server.pid; bundle exec rails s -p 3000"
+    command: bash -c "rm -f /mastodon/tmp/pids/server.pid; bundle exec rails db:migrate ; bundle exec rails db:seed; bundle exec rails s -p 3000"
```

Comment:
I'd like to migrate db manually, because I know when a migration is necessary (Mastodon dashboard tells admins actually), and it's inconvenient to do migration every time when restarting the service.

Reply:
How would this work with the requirement in v3.3 and 3.4 to migrate in multiple steps? Like

1. `docker-compose run --rm -e SKIP_POST_DEPLOYMENT_MIGRATIONS=true web rails db:migrate`
2. clear cache (tootctl)
3. `docker-compose run --rm web rails db:migrate`

Reply:
@mashirozx , @marians, : I m ready to remove that auto migration thing if any maintainer / reviewer  of this project instruct me to do so.

I don't have too much experience with mastodon

Reply:
> How would this work with the requirement in v3.3 and 3.4 to migrate in multiple steps? Like
> 
> 1. `docker-compose run --rm -e SKIP_POST_DEPLOYMENT_MIGRATIONS=true web rails db:migrate`
> 2. clear cache (tootctl)
> 3. `docker-compose run --rm web rails db:migrate`

You should do both in order when migrating.